resources/shuttle-static-folder.mdx

```diff
@@ -41,6 +43,60 @@ async fn main(
 
 The parameter name should be changed to match, in this case being `folder = "public"`.
 
+## Caveats
+
+### Static folders in .gitignore
+
+If you happen to be including your static folder in a `.gitignore` file, you will probably hit an error while trying to deploy your Shuttle app.
+
+Shuttle is using the [ignore](https://crates.io/crates/ignore) crate under the hood, and it will ignore all the files and folders included in your `.gitignore` file. This means that the static folder will not be included in the final archive.
+
+Take into account that this will only affect you if you are using the `shuttle deploy` command. If you are using the `shuttle run` command, you will not have any problems.
+
+If you want to overcome this issue, you'll have to add a `.ignore` file to your project and include your static folder in it. The `.ignore` file takes precedence over the `.gitignore` file, so the static folder will be included in the final archive but will be still ignored by git.
+
+For example, if you have a `.gitignore` file like this:
+
+```gitignore
+dist/
+static/
+target/
+```
+
+Then, in order to include the static folder in the final archive, you'll have to create a `.ignore` file like this:
+
+```ignore
+!static/
+```
+
+The team is working in a more simple way to support this scenario, but this will do the trick for now.
```

Comment:
The language is great too! I think we can change this sentence though:
```suggestion
The team is working on a simpler solution for this scenario, but for now this work-around is needed.
```

Reply:
Thanks for the suggestion!